src/pages/App.tsx

```diff
@@ -119,17 +119,17 @@ export default function App() {
                   <FullFeaturesRoute exact strict path="/pools/mine" component={MyPairs} />
                   <FullFeaturesRoute exact strict path="/rewards" component={Rewards} />
                   <FullFeaturesRoute exact strict path="/rewards/:currencyIdA/:currencyIdB" component={Rewards} />
+                  <FullFeaturesRoute exact strict path="/pools/:currencyIdA/:currencyIdB" component={Pair} />
                   <FullFeaturesRoute
                     exact
                     strict
-                    path="/rewards/:currencyIdA/:liquidityMiningCampaignId/singleSidedStaking"
+                    path="/rewards/single-campaign/:currencyIdA/:liquidityMiningCampaignId"
                     component={LiquidityMiningCampaign}
                   />
-                  <FullFeaturesRoute exact strict path="/pools/:currencyIdA/:currencyIdB" component={Pair} />
                   <FullFeaturesRoute
                     exact
                     strict
-                    path="/rewards/:currencyIdA/:currencyIdB/:liquidityMiningCampaignId"
+                    path="/rewards/campaign/:currencyIdA/:currencyIdB/:liquidityMiningCampaignId"
```

Comment:
Good! Maybe `single-sided-campaign`? @0xVenky can chime on this.

Reply:
Yes I can change to `single-sided-campaign` its more accurate, I was trying to make it short 😄